scripts/import-npa/natural_protected_areas.sh

```
@@ -0,0 +1,461 @@
+  [[ $? -ne 0 ]] && exit 1
+  
+  # Find shapefiles
+  local shapefile_list=$(find_shapefiles "$EXTRACT_DIR")
+  [[ $? -ne 0 ]] && exit 1
+  
+  # Convert shapefiles to SQL
+  # local sql_path=$OUTPUT_SQL
+  local sql_path=$(convert_shapefiles_to_sql "$shapefile_list")
+  [[ $? -ne 0 ]] && exit 1
+  
+  if [[ $? -ne 0 ]]; then
```

Shellcheck (0.10.0)

[warning] This $? refers to a condition, not a command. Assign to a variable to avoid it being overwritten.

(SC2319)